File: C/retangulo.c
#include <stdio.h>
#include <math.h>
int main(){
    double base,altura,diagonal;
    printf("digite a base:\n");
    scanf("%lf",&base);
    printf("digite a altura:\n");
    scanf("%lf",&altura);
    diagonal= sqrt (pow(base,2.0) + pow(altura,2.0));
    printf("area = %.4lf\n",base*altura);
    printf("perimetro = %.4lf\n",(2*base)+(2*altura));
    printf("diagonal = %.4lf\n",diagonal);
    return 0;
}

File: Phyton/old/retangulo.py
import math
base = float(input("base do retangulo: "))
altura = float(input("altura do retangulo: "))
area = base*altura
perimetro = 2*(base+altura)
diagonal = math.sqrt(base**2+altura**25)
print(f"AREA = {area:.4f}")
print(f"perimetro = {perimetro:.4f}")
print(f"diagonal = {diagonal:.4f}")